charts/external-dns/templates/deployment.yaml

```diff
@@ -109,9 +113,9 @@ spec:
             {{- toYaml .Values.livenessProbe | nindent 12 }}
           readinessProbe:
             {{- toYaml .Values.readinessProbe | nindent 12 }}
-          {{- if or .Values.secretConfiguration.enabled .Values.extraVolumeMounts }}
+          {{- if or .Values.secretConfiguration.mountPath .Values.extraVolumeMounts }}
```

Comment:
```suggestion
          {{- if or .Values.secretConfiguration.enabled .Values.extraVolumeMounts }}
```

Reply:
If user has mountPath configured and enabled set to false, it may break his setup.

Reply:
I understand. It makes sense.